content/webapp/pages/concepts/[conceptId].tsx

```diff
@@ -339,7 +339,7 @@ export const ConceptPage: NextPage<Props> = ({
 
       {/* Images */}
       {hasImages && (
-        <ConceptImages as="section">
+        <ConceptImages as="section" aria-label="Images">
```

Comment:
Aria labels should be used for [interactive elements that don't have an good/visible accessible name](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label). Was this for a selector? We could look at other options, our last resort one is to use `data-testid`.